File: ui/src/store/models/MaterialMCQ.ts
import { TrainingMaterial } from './Material';

export interface    MaterialMCQ extends TrainingMaterial {
    question_id: number;
    material_id: number;
    question_text: string;
    answer_1: string;
    answer_2: string;
    answer_3: string;
    answer_4: string;
    correct_1: boolean;
    correct_2: boolean;
    correct_3: boolean;
    correct_4: boolean;
}

File: appServer/DestinyLimoServer/Models/MaterialMCQ.cs
namespace DestinyLimoServer.Models
{
    public class MaterialMCQ : Material
    {
        public int question_id { get; set; }
        public string? question_text { get; set; }
        public string? answer_1 { get; set; }
        public string? answer_2 { get; set; }
        public string? answer_3 { get; set; }
        public string? answer_4 { get; set; }
        public bool correct_1 { get; set; } = false;
        public bool correct_2 { get; set; } = false;
        public bool correct_3 { get; set; } = false;
        public bool correct_4 { get; set; } = false;
    }
}